external/CMakeLists.txt

```
@@ -61,8 +61,8 @@ add_cmake_dependency(
 
 # --------------------------------------------------------------------------------------------------
 # protobuf
-set(PROTOBUF_TAG v25.2)
-set(PROTOBUF_VERSION 4.25.2)
+set(PROTOBUF_TAG v27.2)
+set(PROTOBUF_VERSION 4.27.2)
```

```suggestion
set(PROTOBUF_VERSION 5.27.2)
```
I think is correct here...

---

great catch, changed